### PR TITLE
[controller][samza][vpj][router][fast-client][server][build][doc] Remove non-inclusive terms where possible

### PR DESCRIPTION
**Pull request by nisargthakkar**
## Remove non-inclusive terms where possible

1. Remove some non-inclusive language
2. Change `make_tag.py` script to use the `main` branch
3. Changes to gradle functions to use `main` branch and use upstream remote instead of `origin`

## How was this PR tested?

Internal CI passed

## Does this PR introduce any user-facing changes?

- [ ] No. You can skip the rest of this section.
- [x] Yes. Make sure to explain your proposed changes and call out the behavior change.
    - Some non-inclusive configs have been removed
    - make_tag and gradle scripts will operate against primary venice repo's `main` branch instead of `origin/master`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -417,7 +418,7 @@ subprojects {
 }
 
 spotless {
-  ratchetFrom 'origin/master'
+  ratchetFrom "${git.getUpstreamRemote()}/main"
```
Comment (anaberezhnov) [inline build.gradle:421]: It should be `origin/main` since it runs on every commit, so no need to diff against upstream.

**Review by anaberezhnov — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (nisargthakkar) [reply build.gradle:421]: Should I use `"origin/${git.getBranch()}"` instead?

**Review by anaberezhnov — CHANGES_REQUESTED** [review]
We want to run spotless against `origin/main` specifically. `git.getBranch()` returns current branch, which is usually not the main branch.

File: build.gradle (hunk 1 above)
Comment (nisargthakkar) [reply build.gradle:421]: Discussed offline. We agreed to keep using `"${git.getUpstreamRemote()}/main"`

I tried changing it to: `"${git.upstreamRemote}/main"` but it didn't work

**Review by anaberezhnov — DISMISSED** [review]

**Review by anaberezhnov — DISMISSED** [review]

**Review by anaberezhnov — APPROVED** [review]

**Pull request merged**